### PR TITLE
[WIP] Add recipe for friendsofsymfony/user-bundle

### PR DESCRIPTION
**Pull request by Devtronic**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Read the documentation: https://symfony.com/doc/master/bundles/FOSUserBundle/index.html
+fos_user:
+#    db_driver: orm # other valid values are 'mongodb' and 'couchdb'
+    firewall_name: main
```
Comment (Pierstoval) [inline friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml]: Instead, it's better IMO to directly uncomment `db_driver`, `firewall_name` and `user_class` in the recipe as they are all mandatory anyway in the bundle.

Comment (Devtronic) [reply friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml]: Thanks, you're right. I changed the config in e129d94

File: friendsofsymfony/user-bundle/2.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "FOS\\UserBundle\\FOSUserBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MAILER_USER": ""
```
Comment (Pierstoval) [inline friendsofsymfony/user-bundle/2.0/manifest.json]: I think `MAILER_USER` is inconsistent with possible configuration value. I think `MAILER_SENDER` would probably be more appropriate. A "user" option is more to configure the mailer with a user.

File: friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+fos_user:
+    db_driver: orm # other valid values are 'mongodb' and 'couchdb'
+    firewall_name: main
+    user_class: AppBundle\Entity\User
+    from_email:
+        address: "%env(MAILER_USER)%"
+        sender_name: "%env(MAILER_USER)%"
+
+services:
+    validator.builder:
```
Comment (Pierstoval) [inline friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml]: That's absolutely not how the recipe should work. Instead, the package itself should require the Symfony package that installs this service, which is `symfony/validator` in this case.

This is a problem from the FOSUserBundle side that should be fixed before a recipe can be created.

Comment (Devtronic) [reply friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml]: uh... I understand, sorry.

File: friendsofsymfony/user-bundle/2.0/manifest.json (hunk 2 above)
Comment (fabpot) [reply friendsofsymfony/user-bundle/2.0/manifest.json]: I agree

File: friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml (hunk 3 above)
Comment (Nyholm) [reply friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml]: This service should be removed. You use add `symfony/validator` to the dependencies section in your composer.json.

**Comment by Nyholm** [comment]
FYI. This is a duplicate of #270.

**Pull request closed without merging**